testsuite/python/interactions_non-bonded_interface.py

```diff
@@ -25,7 +25,12 @@
 
 
 class Non_bonded_interactionsTests(ut.TestCase):
-    system = espressomd.System(box_l=[20.0, 20.0, 20.0])
+    system = espressomd.System(box_l=[30.0, 30.0, 30.0])
```

Comment:
Yes, with `20` on the x-axis the test fails on 3 MPI ranks because the LJ range of 8.4 is too large. This error message never surfaced before because runtime errors were never checked when setting a interaction. It's only when the integrator runs that it throws, which can be really confusing when the integrator is run indirectly, e.g. via P3M tuning...